src/site/antora/modules/ROOT/pages/manual/appenders.adoc

```diff
@@ -212,603 +213,360 @@ See xref:manual/configuration.adoc#lazy-property-substitution[runtime property s
-
-|syncSend
-|boolean
-|The default is `true`, causing sends to block until the record has been acknowledged by the Kafka server. When set to `false`, sends a return immediately, allowing for lower latency and significantly higher throughput. _New since 2.8. Be aware that this is a new addition, and it has not been extensively tested.
-Any failure sending to Kafka will be reported as an error to xref:manual/status-logger.adoc[] and the log event will be dropped (the ignoreExceptions parameter will not be effective).
-Log events may arrive out of order on the Kafka server._
-
-|properties
-|Property[]
-|You can set properties in https://kafka.apache.org/documentation.html#producerconfigs[Kafka producer properties]. You need to set the `bootstrap.servers` property, there are sensible default values for the others. Do not set the `value.serializer` nor `key.serializer` properties.
-|===
+See xref:manual/appenders/database.adoc[] for details.
 
-Here is a sample KafkaAppender configuration snippet:
+[#network-appenders]
+=== Network appenders
 
-[source,xml]
-----
-<?xml version="1.0" encoding="UTF-8"?>
-<Configuration>
-  <!-- ... -->
-  <Appenders>
-    <Kafka name="Kafka" topic="log-test">
-      <PatternLayout pattern="%date %message"/>
-      <Property name="bootstrap.servers">localhost:9092</Property>
-    </Kafka>
-  </Appenders>
-  <!-- ... -->
-</Configuration>
-----
+These appenders use simple network protocols to transmit log events to a remote host.
+The supported network protocols are:
+
+`UDP`::
+`TCP`::
+These are handled by the xref:manual/appenders/network.adoc#SocketAppender[Socket Appender].
+
+`HTTP`::
+This is handled by the xref:manual/appenders/network.adoc#HttpAppender[HTTP Appender].
+
+`SMTP`::
+This is handled by the xref:manual/appenders/network.adoc#HttpAppender[SMTP Appender].
 
-This appender is synchronous by default and will block until the record has been acknowledged by the Kafka server, timeout for this can be set with the `timeout.ms` property (defaults to 30 seconds).
-Wrap with https://logging.apache.org/log4j/2.x/manual/appenders.html#AsyncAppender[Async appender] and/or set syncSend to `false` to log asynchronously.
+See xref:manual/appenders/network.adoc[] for details.
 
-This appender requires the https://kafka.apache.org/[Kafka client library].
-Note that you need to use a version of the Kafka client library matching the Kafka server used.
+[#message-queue-appenders]
+=== Message queue appenders
 
-_Note:_ Make sure to not let `org.apache.kafka` log to a Kafka appender on DEBUG level, since that will cause recursive logging:
+Message queue appenders forward log events to a message broker to provide guaranteed delivery over the network.
```

Comment:
_"to provide guaranteed delivery over the network"_ is an appender-specific feature that we should not generalize for the group:

```suggestion
Message queue appenders forward log events to a message broker.
```